src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -345,14 +346,17 @@ export class PanelComponent extends Disposable implements IThemable {
 
 	private focusNextTab(): void {
 		const currentIndex = this.focusedTabHeaderIndex;
+		this._tabHeaders.toArray()[currentIndex].nativeElement.tabIndex = -1;
```

Comment:
Aren't you already doing this in the template?

`[attr.tabindex]="tab.active ? '0': '-1'"`

Reply:
This is called when we use arrow keys to change focus. We do not change the active tab in that case.

Reply:
Yeah, I agree with @alanrenmsft. We should be updating the active state when the active tab changes regardless of whether it's through the arrow keys, clicking or whatever. Is that not happening or something?